projects/Mallard/jest-setup.js

```diff
@@ -1,3 +1,3 @@
 jest.mock('src/helpers/locale', () => ({
-    locale: 'en_GB',
+    locale: 'en_US',
```

Comment:
Changed to be different to UK which is our "default edition" if the local is UK or different to US and AUS. This will help find more bugs in tests